fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -1532,36 +1533,17 @@ ACTOR Future<Optional<std::string>> globalConfigCommitActor(GlobalConfigImpl* gl
 		if (entry.first) {
 			if (entry.second.present() && iter->begin().startsWith(globalConfig->getKeyRange().begin)) {
 				Key bareKey = iter->begin().removePrefix(globalConfig->getKeyRange().begin);
-				vh.mutations.emplace_back_deep(vh.mutations.arena(),
-				                               MutationRef(MutationRef::SetValue, bareKey, entry.second.get()));
-
-				Key systemKey = bareKey.withPrefix(globalConfigKeysPrefix);
-				tr.set(systemKey, entry.second.get());
+				insertions.push_back_deep(insertions.arena(), KeyValueRef(bareKey, entry.second.get()));
```

Comment:
I think the memory here will remain valid through the call to `applyChanges`, in which case we could do a non-copying append using `push_back()`.

Reply:
I tried this (I just changed `push_back_deep` to `push_back`) and it actually ended up breaking things, causing insertions to never take place. So I kept it as `push_back_deep`.